docs/machine-learning/resources/transforms.md

```diff
@@ -0,0 +1,131 @@
+# Machine learning transforms
+
+The following tables contains all of the supported data transforms in ML.NET and a description of their functionality.
+
+Click to jump immediately to a certain section of transforms:
+* [Categorical](#categorical)
+* [Combiners and segregators](#combiners-and-segregators)
+* [Feature selection](#feature-selection)
+* [Featurizers](#featurizers)
+* [Label parsing](#label-parsing)
+* [Missing Values](#missing-values)
+* [Normalization](#normalization)
+* [Row filters](#row-filters)
+* [Schema](#schema)
+* [Text processing and featurization](#text-processing-and-featurization)
+* [Miscellaneous](#miscellaneous)
+
+> [!NOTE]
+> ML.NET is currently in Preview. Not all data transforms are currently supported. To submit a request for a certain transform, please open an issue in the [dotnet/machinelearning repository](https://github.com/dotnet/machinelearning/issues).
+
+## Categorical
+
+| Transform | Definition |
+| --- | --- |
+| CategoricalHashOneHotVectorizer | Encodes the categorical variable with hash-based encoding. |
+| CategoricalOneHotVectorizer | Encodes the categorical variable with one-hot encoding based on term dictionary. |
+
+## Combiners and segregators
+
+| Transform | Definition |
+| --- | --- |
+| CombinerByContiguousGroupId | Groups values of a scalar column into a vector, by a contiguous group ID. |
+| FeatureCombiner | Combines all the features into one feature column. |
+| ManyHeterogeneousModelCombiner | Combines a sequence of TransformModels and a PredictorModel into a single PredictorModel. |
+| ModelCombiner	| Combines a sequence of TransformModels into a single model. |
+| Segregator | Un-groups vector columns into sequences of rows, inverse of Group transform. |
```

Comment:
Un-groups --> ungroups
rows, inverse  --> rows; the inverse 
Group transform isn't a transformation listed in this topic; should it be?